### DISCUSSION TITLE
Amazon Q modifies dotfiles even if you don't grant permission

### DISCUSSION THREAD

**Discussion by injust** [aws/codewhisperer-command-line-discussions#258] [Known bugs]
As of version 1.4.4, Amazon Q asks for permission before installing shell integrations. But before you get a chance to click Install/Skip, it already modifies your dotfiles.

Amazon Q installed itself to:
* `~/.config/fish/conf.d/00_fig_pre.fish`
* `~/.config/fish/conf.d/99_fig_post.fish`
* VS Code extension

Could you like...not do this?

**Comment by wojtekgalaj** [comment]
Well that's nice. Took me entirely too long to figure out why my new alias was missing until I figured out my .profile .bashrc and more have been oh so kindly adjusted by the immensly helpful installer script. As with injust's case, no prompt was given but I'm sure that is just not to bother me as I try the 25th prompt to get that sam cli argument set right.

Uninstalled for now but hey, if you decide not to take control over my, sort of important, files, give us a shout.
